conanfile.py

```diff
@@ -1,52 +1,66 @@
-"""Conan recipe for LKSCTP Tools
-"""
-import os
-from tempfile import mkdtemp
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os, glob, shutil
 from conans import ConanFile, AutoToolsBuildEnvironment, tools
 
 
 class LKSCTPToolsConan(ConanFile):
-    """Download LKSCTP Tools, build and create package
-    """
     name = "lksctp-tools"
     version = "1.0.17"
-    generators = "cmake", "txt"
+    generators = "cmake"
     settings = "os", "arch", "compiler", "build_type"
-    options = {"shared": [True, False]}
-    default_options = "shared=False"
-    url = "https://github.com/uilianries/conan-lksctp-tools"
+    options = {"shared": [True, False], "fPIC": [True, False], "with_sctp": [True, False]}
```

Comment:
I'd prefer `with_sctp` with default false. IMHO it is way too misleading to add a conan package to your project and automagically in the next run your TCP sockets are suddenly SCTP sockets.

Reply:
It's false, look at line https://github.com/bincrafters/conan-lksctp-tools/pull/3/files#diff-2b1d42f71f22b7ea0412d7602dec166fR14.

`options` only set the allowed values, but doesn't fill the default one.  
The default value is filled by `default_options`

Reply:
Forgot about that, sorry!